ext/oci8/oci8.stub.php

```diff
@@ -0,0 +1,826 @@
+ * @alias oci_commit
+ * @deprecated
+ */
+function ocicommit($connection_resource): bool {}
+
+/**
+ * @param resource $statement_resource
+ * @return string|false|null @todo return false, null?
+ */
+function oci_field_name($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ * @return string|false|null @todo return false, null?
+ * @alias oci_field_name
+ * @deprecated
+ */
+function ocicolumnname($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ * @return int|false|null @todo return false, null?
+ */
+function oci_field_size($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ * @return int|false|null @todo return false, null?
+ * @alias oci_field_size
+ * @deprecated
+ */
+function ocicolumnsize($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ * @return int|false|null @todo return false, null?
+ */
+function oci_field_scale($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ * @return int|false|null @todo return false, null?
+ * @alias oci_field_scale
+ * @deprecated
+ */
+function ocicolumnscale($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ * @return int|false|null @todo return false, null?
+ */
+function oci_field_precision($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ * @return int|false|null @todo return false, null?
+ * @alias oci_field_precision
+ * @deprecated
+ */
+function ocicolumnprecision($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ * @return string|int|false|null @todo return false, null?
+ */
+function oci_field_type($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ * @return string|int|false|null @todo return false, null?
+ * @alias oci_field_type
+ * @deprecated
+ */
+function ocicolumntype($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ * @return int|false|null @todo return false, null?
+ */
+function oci_field_type_raw($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ * @return int|false|null @todo return false, null?
+ * @alias oci_field_type_raw
+ * @deprecated
+ */
+function ocicolumntyperaw($statement_resource, mixed $column_number_or_name) {}
+
+/**
+ * @param resource $statement_resource
+ */
+function oci_field_is_null($statement_resource, mixed $column_number_or_name): bool {}
+
+/**
+ * @param resource $statement_resource
+ * @alias oci_field_is_null
+ * @deprecated
+ */
+function ocicolumnisnull($statement_resource, mixed $column_number_or_name): bool {}
+
+function oci_internal_debug(bool $mode): void {}
```

Comment:
I can't see any parameters. (?)

Reply:
Shouldn't this be deprecated? It doesn't seem to be very useful anymore.

Reply:
I've spoken to @cjbj about this and i will make a PR to remove this function after this one has been merged.